bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -709,7 +704,7 @@ func (r *KubeadmConfigReconciler) reconcileDiscovery(ctx context.Context, cluste
 		hashes, err := certificates.GetByPurpose(secret.ClusterCA).Hashes()
 		if err != nil {
 			log.Error(err, "Unable to generate Cluster CA certificate hashes")
-			return err
+			return ctrl.Result{}, err
```

Comment:
Wrap error to provide more context?

Reply:
The error seems it's already wrapped in the `.Hashes()` method

Reply:
thanks for double checking